android/app/build.gradle

```diff
@@ -23,9 +23,17 @@ android {
     }
     buildTypes {
         debug {
+            buildConfigField("String", 'OAUTH_BASE_URL', '"http://10.0.2.2:8080/auth/"')
```

Comment:
It seems you committed with a different username/credential as the one that openned this PR. I think that might bring up problems later. Kindly confirm if this was on purpose